src/helpers/purchase-operation-helper.ts

```diff
@@ -129,7 +129,6 @@ export class PurchaseOperationHelper {
     productId: string,
     purchaseOption: PurchaseOption,
     presentedOfferingContext: PresentedOfferingContext,
-    email?: string,
```

Comment:
@vicfergar to review

Reply:
It’s fine to remove this parameter if we want to maintain a simpler interface.